### ISSUE TITLE
Beatmap rankings page design could use a lot of love.

### ISSUE THREAD

**Issue by VeilStar** [ppy/osu-web#2015]
Compared to pretty much anything else on the website the beatmap rankings page design looks pretty damn bad bad, and feels quite out of place. I couldn't find any open issues on this, which makes me wonder if I've gone blind.

To start off, why does it not use a very similar design to the performance/score/whatever rankings pages such as https://osu.ppy.sh/rankings/osu/performance ? I would imagine you'd want the beatmap ranking pages be similar. Having 'lines' between each score, clearly visible columns with proper labels at the top, grayscale text (instead of freaking blue for some reason), and actually aligned information feels so much nicer. (And the darker theme is a nice bonus.)

On the beatmap ranking pages the "In the Lead" and "Your Best" scores have a ridiculous amount of white space for no apparent reason.
https://puu.sh/yIgSq/c4d143aacd.jpg
The top section of the panel is by no means small and holds very little information. Sure, it's the number 1 score, make it stand out, but a design much more similar to your score if it's in the top 50 (not the "Your Best" score but this https://puu.sh/yIgST/e4f2630d98.jpg) looks a lot more fitting. Especially given the information density about the actual beatmap at the top of the page (which is right above it).

To continue from that, the mods, score, accuracy, combo, hitvalues, and misses for the different panels ("In the Lead", "Your Best", your score in the top 50, and all other scores shown) are all aligned with no regards to each other. While you do want the text to be different sizes, I'm sure alignment of this could be done MUCH better. http://puu.sh/yIh5k/30c485cf24.jpg Yeah uh.. this looks like a mess.

Edit: https://github.com/ppy/osu-web/issues/559 has a bunch of improvements over what is currently implemented (though imho still not enough), but is closed.

**Comment by peppy** [comment]
A new design is already well in progress to fix these things (and this has been addressed in other issues in more localised points). Your prose is really hard to parse, filled with negative language and not so useful since we are already aware of concerns.

Closing with a "we are aware".
